lib/services/local/userpreferences.go

```diff
@@ -142,36 +153,34 @@ func createBackendItem(username string, preferences *userpreferencesv1.UserPrefe
 	return item, nil
 }
 
-// mergePreferences merges the values from src into dest.
-func mergePreferences(dest, src *userpreferencesv1.UserPreferences) {
-	if src.Theme != userpreferencesv1.Theme_THEME_UNSPECIFIED {
-		dest.Theme = src.Theme
-	}
+// overwriteValues overwrites the values in dst with the values in src.
+func overwriteValues(dst, src protoreflect.ProtoMessage) error {
```

Comment:
can you please add a comment mentioning that it only works because proto.range only visits non-nil/non-empty fields? 

It would become clear to anyone using it how it works.

Reply:
Added